### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by ImKing-github** [Koupah/Among-Us-Editor#52]
When u try to use in the new version, it don't change in-game

**Comment by Koupah** [comment]
What doesn't work about it?

Are you closing your game, using the Editor then opening your game again?

**Comment by ImKing-github** [comment]
yes, to be exact, when I try to use the editor and reopen the game, it goes back to the last skin used before using the editor

Obter o Outlook para Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by WaddlesPlays** [comment]
I have the same issue, where the editor does not make changes ingame. But if I carry the values the editor changes in playerPrefs into (random numbers and letters)_playerPrefs, the changes show up in game. Might be worth mentioning I play on the Epic Games version and I am logged in ingame. Not sure if this helps or not.

**Comment by Koupah** [comment]
> I have the same issue, where the editor does not make changes ingame. But if I carry the values the editor changes in playerPrefs into (random numbers and letters)_playerPrefs, the changes show up in game. Might be worth mentioning I play on the Epic Games version and I am logged in ingame. Not sure if this helps or not.

From the sounds of it, the epic games version uses another file for storing preference. I'll look into this.

**Comment by tnhung2011** [comment]
My Among Us Editor not work too. Looks like you didn't update the game since April.

**Comment by tnhung2011** [comment]
I use non-Steam version too :|

**Comment by tnhung2011** [comment]
A few months earlier, I opened Among Us Editor and it didn't open. I tryed to re-install then opened it and... it didn't open, again.

**Comment by tnhung2011** [comment]
I used itch.io version :(
